saleor/dashboard/category/forms.py

```diff
@@ -0,0 +1,24 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from mptt.forms import TreeNodeChoiceField
+
+from ...product.models import Category
+
+
+class CategoryForm(forms.ModelForm):
```

Comment:
From the code it seems that this form is used for editing existing categories as well as creating new ones. If so, you may want to use `category.move_to(…)` or extend `mptt.forms.MoveNodeForm`.